licensecheck.properties

```diff
@@ -0,0 +1,25 @@
+category=Languages
+name=License Check
+description=Checks Maven, Gradle and NPM project for license violations
+issueTrackerUrl=https://github.com/porscheinformatik/sonarqube-licensecheck/issues
+devVersion=5.0.0-beta-2
```

Comment:
In fact, there's no point in listing a devVersion. This used to be used by internal tooling but is purely vestigial now.

Reply:
Removed it